### DISCUSSION TITLE
Setting `aria-disabled="true"` with b-button and <button>

### DISCUSSION THREAD

**Discussion by jakedolan** [bootstrap-vue/bootstrap-vue#6678] [Q&A]
Is there any way to set `aria-disabled="true"` for the b-button component while keeping it as an html `<button>`? 

I see that some other aria-* attributes are passed through as attributes to the generated `<button>` tag, but not `aria-disabled`. 

The ability to set the property `disabled` is not interchangeable with `aria-disabled="true"` as `disabled` removes the element from being perceivable. In some use cases, we should be able to inform assistive technology that a button is disabled while keeping it perceivable. 

I recognize that this requires the developer to ensure that the button is visually and functionally disabled so it requires more intention with its use.

From the documentation if you set the `disabled` property and  `href="#"` property (changing it the button to `<a role="button">`) it will add the `aria-disabled="true"`, but it also adds the attribute `tabindex="-1"`.

I am curious if I am missing some way to set it directly.

If not and there isn't a good reason why it isn't being passed through, I would be interested in investigating it for a pull request.

**Comment by phegman** [comment]
Also seeing this, opened https://github.com/bootstrap-vue/bootstrap-vue/pull/7190
